orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -10,28 +10,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO move to preburn
```

Comment:
Why was an extra step added?

Reply:
The step on master to remove the existing registries using `/etc/apt/sources.list.d/*.list` does not work. The Ansible file module does not support wildcard expressions. Compare for example the outputs of the runs on [master](https://github.com/magma/magma/actions/runs/3620935852/jobs/6103757688#step:10:105) and [my fork](https://github.com/mpfirrmann/magma/actions/runs/3629998442/jobs/6122841370#step:10:109).